app/views/claims/information_provided.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">How we will use the information you provide</h1>
+    <p class="govuk-body">
+      To process your claim we will check the information you provide against the information the
+      Department for Education hold.
+    </p>
+
+    <p class="govuk-body">To determine if your claim is valid we will:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        check your eligibility against the Database of Qualified Teachers. You may have used with
+        this in the past as the Teacher Self-Service Portal
+      </li>
+      <li>check your eligibility against the Teachers Pension Service</li>
+      <li>check your eligibility with any previous or current schools you have worked for</li>
+    </ul>
+
+    <p class="govuk-body">To progress and pay your claim we will:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>pay you as the Department for Education. You will see this on a future P60</li>
+      <li>send some of your personal information to HMRC</li>
+      <li>pay a tax, student loan and national insurance contribution to HMRC</li>
+    </ul>
+
+    <p class="govuk-body">
+      We will send you details about these payments once your application is complete.
+    </p>
+
+    <%= link_to "Continue to GOV.UK Verify", new_verify_authentications_path, class: "govuk-button" %>
+
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          What information is sent to HMRC?
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+        We send HMRC your full name, date of birth, payment amount and gender to ensure your tax
```

Comment:
"payment amount"? What does that refer to? Could it be clarified?

Reply:
Yes, it could be. Again, this is hastily put-together content and a stop-gap solution that hasn't had a content designer go over it. We know this page needs work, let's get this out to unblock private beta and revisit this page properly next.